entries/k/klantportaal.digireceptie.nl.json

```
@@ -0,0 +1,11 @@
+{
+  "DigiReceptie Klantportaal": {
+    "domain": "klantportaal.digireceptie.nl",
```

The value for the domain tag should be the domain for the service (like a homepage) and not link to the login page.
I suggest you use digireceptie.nl instead.
```suggestion
    "domain": "digireceptie.nl",
```